server/src/validators/commentValidator.ts

```diff
@@ -0,0 +1,21 @@
+import { Validator } from "../types";
+
+const commentCreateValidator: Validator[] = [
+  {
+    key: "comments",
+    required: true,
+    type: "string",
+  },
+  {
+    key: "itemId",
+    required: true,
+    type: "string",
```

Comment:
both ids should be of type uuid, I've added the logic for it in my changes, just change it so it will be able to validate that the user sends a valid uuid